Wootric/WootricSDK-iOS
# Bug: fix 'User-Agent'

This PR sets the correct HTTP Header with 'Wootric-Mobile-SDK'

## Changes
- Update WTRAPICLient

## How to test
1- Pull branch
2- Set the correct credentials in ViewController.m
3- Try enabling/disabling web switch and mobile swith in your account's settings. It should work as expected

## Trello
https://trello.com/c/4iKDvIGd/1611-ios-sdk-doesn-t-turn-on-when-the-dashboard-ui-switches-are-configured-for-only-mobile-on-web-must-also-be-on-for-survey-to-fire-
